app/views/client_applications/show.html.haml

```diff
@@ -40,6 +40,11 @@
       %tr
         %th.text-right Roles
         %td#roles= @client_application.roles.join(', ')
+    -if @client_application.api_version == 1
```

Comment:
Fun!

Since the ClientApplication is ONLY accessible in Gravity V2 I dont easily have access to see if a given `@client_application` actually has any `client_application_partner` joins. I'm just keeping off of the client app's API version (this should only be offered for V1 partner API clients)

Reply:
Seems fine!